File: BackEnd/Zalo_mini/Zalo_mini/Models/group_member.cs
using System;
using System.Collections.Generic;

namespace Zalo_mini.Models
{
    public partial class group_member
    {
        public long id { get; set; }
        public long group_id { get; set; }
        public long user_id { get; set; }
        public string? role { get; set; }
        public DateTime? joined_at { get; set; }
        public bool? is_active { get; set; }

        public virtual chat_group group { get; set; } = null!;
        public virtual user user { get; set; } = null!;
    }
}

File: BackEnd/java/ZaloMini/src/main/java/com/devmam/zalomini/entities/GroupMember.java
package com.devmam.zalomini.entities;

import jakarta.persistence.*;
import lombok.*;
import org.hibernate.annotations.ColumnDefault;
import org.hibernate.annotations.OnDelete;
import org.hibernate.annotations.OnDeleteAction;

import java.time.Instant;

@Builder
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Entity(name = "GroupMember")
@Table(name = "group_members")
public class GroupMember {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @OnDelete(action = OnDeleteAction.CASCADE)
    @JoinColumn(name = "group_id", nullable = false)
    private ChatGroup group;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @OnDelete(action = OnDeleteAction.CASCADE)
    @JoinColumn(name = "user_id", nullable = false)
    private User user;

    @ColumnDefault("'member'")
    @Lob
    @Column(name = "role")
    private String role;

    @ColumnDefault("CURRENT_TIMESTAMP")
    @Column(name = "joined_at")
    private Instant joinedAt;

    @ColumnDefault("1")
    @Column(name = "is_active")
    private Boolean isActive;

}